pydrive2/files.py

```diff
@@ -518,6 +518,30 @@ def Delete(self, param=None):
         """
         self._FilesDelete(param=param)
 
+    @LoadAuth
+    def Copy(self, target_folder: "GoogleDriveFile", new_title = None):
```

Comment:
should be an extra argument: `param = None`

let's adhere to the existing code and not do `: "GoogleDriveFile"`

see if the new_title can be passed as part of the `params`

Reply:
If i understood the use of `param` correctly we should have:

```python
self.auth.service.files().copy(**param).execute()
``` 

in this case `new_title` is a part of `body` that is a part of `param`, so the user would have to pass a double dict like:

```python
param = {
    "body": {
        "title": new_title
    }
}
```

and than we fill the other fields inside the method.

This will also leave the possibility for the user to pass any other parameters. 
What do you think? if is just for the `new_title` i feel like is a little over engineered

Reply:
check the `InsertPermission` please - it can be probably done this way:

`def Copy(self, target_folder, new_title, param=None):`

param argument is still needed, it allows user to pass some extra configs if needed

Reply:
solved in [327e2eb](https://github.com/iterative/PyDrive2/pull/188/commits/327e2eba0f6b4680d05b045f6139ad5f85320fa0)